### ISSUE TITLE
Unity cloud build ios fails with error after updating casai to 3.5.2 from 3.4.2

### ISSUE THREAD

**Issue by fzfking** [cleveradssolutions/CAS-Unity#5]
Fails with error: [error] [2024-02-09T15:59:49.430Z] - 7.3.35.2.7.4 - INFO: ▸ ❌  error: Multiple commands produce '/BUILD_PATH/Library/Developer/Xcode/DerivedData/Unity-iPhone-bvukkiuohmdwhfgxfrghubxkprlu/Build/Intermediates.noindex/ArchiveIntermediates/Unity-iPhone/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/UnityFramework.framework/PrivacyInfo.xcprivacy'
Maybe there is workaround? We are using: Cas.ai, Pollfish, Firebase, Amplitude, Appsflyer, Tapjoy, Unity Analytics

**Comment by Str4tos** [comment]
Hi @fzfking, thank you for raising the issue.  
We are aware of this problem, however, it is not within our control. The issue stems from an update in Unity where they began adding `PrivacyInfo.xcprivacy` to the project, and due to the incorrect declaration of the `PrivacyInfo.xcprivacy` file for Yandex Ads. 

We are already in communication with Yandex Ads to resolve their issue. 
For now, you can fix the problem by following these steps in the Xcode project:
1. Open the file through Finder in the Xcode project folder.
```
Pods/Target Support Files/Pods-UnityFramework/Pods-UnityFramework-resources.sh
```
2. Find all the following lines and remove them.
```
install_resource "${PODS_ROOT}/YandexMobileAds/PrivacyInfo.xcprivacy"
```
3. Save the file.
4. Open the Xcode project.
5. Use the `Product > Clean Build Folder` menu.
6. Build the project again.

We will try to fix this conflict by the next update of the CAS Unity Plugin.
Thank you.

**Comment by fzfking** [comment]
We are using Unity Cloud build and cant open xcode project (dont have mac with xcode anyway). Is there is any solution with PostProcessingScript? Or for now we will stay at 3.4.2 and will be wait your's next update when this issue will be fixed?

**Comment by Str4tos** [comment]
Then you will have to wait for the update of the CAS Unity Plugin. We are working on publishing it as soon as possible.

**Comment by Str4tos** [comment]
It will be fixed in the CAS Unity Plugin 3.6.0 along with the Yandex Mobile Ads SDK 6.4.0 update.  
Thank you for your patience.
